CHANGELOG.md

```diff
@@ -1,5 +1,7 @@
 # Upcoming
 
+* Fix message render for non TextEditor panes
```

Comment:
This message makes it sound like something was broken. I think a better message would be "Hide messages when non-TextEditors (ex. Settings) are active"